### ISSUE TITLE
LocalClient download/upload path issues

### ISSUE THREAD

**Issue by jasonsebastian** [kwinata/pacco#8]

**Comment by kwinata** [comment]
@jasonsebastian Please give minimum reproducible script that gives that error. Thanks!

**Comment by jasonsebastian** [comment]
```
mkdir a && cd a && echo 'export LIBS_PATH="$(pwd)/libs"; pushd b; source b.sh; popd' > a.sh && chmod +x a.sh
mkdir b && cd b && echo 'pacco download openssl-gto ${LIBS_PATH} os=android version=2.1.0 obfuscation=obfuscated' > b.sh && chmod +x b.sh
cd .. && mkdir libs # the existing libs folder is the suspected problem
./a.sh
```

**Comment by kwinata** [comment]
Fixed in 0.1.6
